src/main/java/com/uid2/shared/store/IKeysetProvider.java

```diff
@@ -0,0 +1,8 @@
+package com.uid2.shared.store;
+
+import java.time.Instant;
+
+public interface IKeysetProvider {
```

Comment:
Do we actually need this interface? RotatingKeysetProvider implements it but IKeysetProvider doesn't seem to be used anywhere else

Reply:
I was just copying how ACLs work. Looks like operator uses it but doesn't implement it, so I think we are probably good to remove it and just use the RotatingKeysetProvider